src/viam/rpc/dial.py

```diff
@@ -314,5 +314,5 @@ async def dial_direct(address: str, options: Optional[DialOptions] = None) -> Ch
     return await _dial_direct(address, options)
 
 
-async def _dial_app(options: DialOptions) -> Channel:
-    return await _dial_direct("app.viam.com:443")
+async def _dial_app(app_url: str) -> Channel:
+    return await _dial_direct(app_url)
```

Comment:
[q] I see that above in this file we do:
```
    host, port = _host_port_from_url(address)
    if not port:
        port = 80 if insecure else 443
    server_hostname = host
```
Should we maybe have it default to `8080`? Were you able to test this both with `localhost:8080` as well as `http://localhost`? I am not sure if it would work for the latter option as-is but haven't tested. We could potentially match on the string `localhost` (rather than requiring the `http://` prefix) and hardcode 8080 in that case.

Reply:
`localhost:8080` definitely works, so does `http://localhost` if I update the default to be 8080. I think we kept it as `80` because that's default insecure port

Reply:
ok! I'm not sure I follow why keeping it as `80` as desirable but I'm sure I'm lacking a lot of context here. Going to hit approve. Thank you!